Package.swift

```diff
@@ -4,7 +4,7 @@
 import PackageDescription
 
 let package = Package(
```

Comment:
You are probably going to want to duplicate this as `Package@swift-6.0.swift` with `// swift-tools-version: 6.0` at the top to enable strict concurrency checking with explicit swift 6 support.

Reply:
I'll give that a shot right after we get this reviewed and merged, but NFTPR.